doc/en/conf.py

```diff
@@ -112,16 +112,18 @@
 # A list of ignored prefixes for module index sorting.
```

Comment:
Please do not use mentions in commit messages.

Reply:
sorry for that - will the gone one the commit is squash-merged.

Reply:
You need to force-push then, squash-merging is not enabled here.. :/

Reply:
..but you can use `[ci skip]` then in the first line of the commit message then probably.

Reply:
Any objections against me squahing all commits onto one and force pushing that one?

Reply:
No.. :)

Reply:
Done. Is there anything else missing from your point of view?